dataprocessors/json/json.go

```diff
@@ -36,7 +36,7 @@ type ValidationError struct {
 }
 
 func (e *ValidationError) Error() string {
```

Comment:
As this function is called `Error()` would it make sense to return an `error` object with `fmt.Errorf()` instead of a string?

Reply:
This implements the `error` interface

https://www.digitalocean.com/community/tutorials/creating-custom-errors-in-go